src/SolutionTemplate/UnoSolutionTemplate.WinUI.net6/Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj

```diff
@@ -21,7 +21,9 @@
 		<PackageReference Include="Uno.WinUI.Skia.Gtk" Version="4.2.6" />
 		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.2.6" Condition="'$(Configuration)'=='Debug'" />
 		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="3.0.5" />
-		<PackageReference Include="Uno.UI.Lottie" Version="4.5.9" />
+		<PackageReference Include="Uno.WinUI.Lottie" Version="4.5.9" />
+		<PackageReference Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.1-preview.79" />
+		<PackageReference Include="SkiaSharp.Skottie" Version="2.88.1-preview.79" />
```

Comment:
Push SkiaSharp.Skottie to a stable version eg 2.88.1 or 2.88.2

Reply:
Will do in a separate PR as it's not related to this PR specific change.